src/pages/home/home.ts

```diff
@@ -7,8 +7,53 @@ import { NavController } from 'ionic-angular';
 })
 export class HomePage {
 
+  users: any
+  user: any
   constructor(public navCtrl: NavController) {
```

Comment:
Looks like you are only using the ```users: any``` variable and not the ```user: any```

Reply:
good catch, deleted.